src/cirrus/release.py

```diff
@@ -19,7 +19,7 @@
 from cirrus.documentation_utils import (
     build_docs,
     build_doc_artifact,
-    upload_documentation
+    publish_documentation
```

Comment:
I would just leave this out of the release command:
The release command builds and uploads releases, the docs command builds and uploads docs, I dont think it is too much to ask a user to do:

git cirrus release build
git cirrus docs build 
git cirrus release upload 
git cirrus docs publish 

as part of their release process. (The release command is already one of the more complicated bits of logic as is, so would prefer to keep things atomic there)